darwin/importer/importer.py

```diff
@@ -1035,15 +1009,17 @@ def _handle_subs(
     return data
 
 
-def _handle_complex_polygon(
+def _format_polygon_for_import(
     annotation: dt.Annotation, data: dt.DictFreeForm
 ) -> dt.DictFreeForm:
-    if "complex_polygon" in data:
-        del data["complex_polygon"]
-        data["polygon"] = {
-            "path": annotation.data["paths"][0],
-            "additional_paths": annotation.data["paths"][1:],
-        }
+    if "polygon" in data:
+        if len(annotation.data["paths"]) > 1:
+            data["polygon"] = {
+                "path": annotation.data["paths"][0],
```

Comment:
When importing polygons, they have to be structured as a primary path, and any subsequent paths have to be passed in `additional_paths` in the payload